README.md

```diff
@@ -1,161 +1,46 @@
-# Simplechart
+# ![Simplechart](docs/_includes/logo.png)
 
-## Build process
+Quickly create and structure interactive charts for use in a content management system (CMS).
 
-`$ npm run build` will compile both the App and Widget bundles, using the latest Git commit hash as a version.
+*See [Simplechart for WordPress](https://github.com/alleyinteractive/wordpress-simplechart) for the WordPress plugin.*
 
-_Don't use this_ during normal development. You'll only need to build if you're pushing to an implementation of Simplechart, e.g. its [WordPress plugin](https://www.github.com/alleyinteractive/wordpress-simplechart).
+## Quick Start
+You'll need just a couple things installed before running Simplechart.
 
-## Development mode
+- Node
+- NPM
 
-`$> npm start` then...
+You can confirm these packages are installed by running `node -v` or `npm -v` from your terminal.
 
-### App
+1. Clone the Simplechart repository to your local machine using `git clone git@github.com:alleyinteractive/simplechart.git`.
+2. Change directory to your newly created folder using `cd simplechart`.
+3. Install the Node packages listed in *package.json* by running `npm install`.
+4. Start the Node app using `npm start`.
 
-The app is where you input data, select a chart type, set up options. Within an iframe, e.g. when used in the WordPress plugin, the app sends data to the parent window via postMessage for saving.
+Congrats! You should now be able to access Simplechart at `http://localhost:8080/index.html`.
 
-You can see the app at `http://localhost:8080/index.html`
+If you're ready to deploy the app, head over to the Wiki for the [Deployment Guide](TK).
 
-### Widget
+## Use Cases
+Simplechart provides the most utility at present when used in conjunction with the WordPress plugin. You don't need to install this package to try the WordPress plugin.
 
-The widget recognizes Simplechart placeholder divs on a page, fetches data, and renders charts.
+Although we have focused on WordPress to date, Simplechart itself is CMS agnostic, and can even be used without a CMS. Simplechart allows you to control your own data, and customize default styling to maintain visual consistency with your brand.
 
-You can find widget examples at `http://localhost:8080/widget.html`
+Simplechart runs on modern web technology – React, WebPack, and ES5 – and Simplechart widgets are **AMP-compatible** out of the box.
 
-## Adding a new chart type
+Simplechart is and always will be free as in beer, and the code is licensed under [GPL-2.0](https://wordpress.org/about/gpl/). 
```

Comment:
Simplechart is ~~and always will be free as in beer, and the code is~~ licensed under